src/active_object.rs

```diff
@@ -1,10 +1,12 @@
 // Implements http://rosettacode.org/wiki/Active_object
-#![allow(unstable)]
+#![feature(std_misc)]
+#![feature(io)]
+#![feature(core)]
```

Comment:
Is it possible to have everything in one `feature` attribute (like `#![feature(std_misc, io, core)]`)? I think it's nice to avoid repeating `feature`.